tensorflow/core/util/gpu_launch_config.h

```diff
@@ -203,6 +203,8 @@ GpuLaunchConfig GetGpuLaunchConfigFixedBlockSize(
 #elif TENSORFLOW_USE_ROCM
   hipError_t err = hipOccupancyMaxActiveBlocksPerMultiprocessor(
       &block_count, func, fixed_block_size, dynamic_shared_memory_size);
+  if (block_count < 1)
+    block_count = 1;
```

Comment:
should we give `LOG(ERROR)` or `LOG(WARNING)` here to indicate there is an illegal value....?

Reply:
This warning would then be displayed once or more for every step More logic could be added to limit the spam, but I don't see much value in having that warning there.

Reply:
At least we can directly know something's wrong on `hipOccupancyMaxActiveBlocksPerMultiprocessor`, then we need to infor hip-runtime team to fix it.

Reply:
yeah, it does feel a little weird masking a potential hip API issue. Those issue should be discovered and fixed elsewhere. This will allow TF testing and enablement to continue.